spec/lib/dor/was_crawl/dissemination/utilities_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Dor::WasCrawl::Dissemination::Utilities do
   describe '.run_sys_cmd' do
     it 'returns nothing with succesful command' do
-      described_class.run_sys_cmd('ls', '')
+      expect { described_class.run_sys_cmd('ls', '') }.not_to raise_error
```

Comment:
To better match the `it` label, I wonder if the expectation here should be that the return value is "nothing" (empty string or `nil` or whatever), rather than testing that it does not raise an error?

Reply:
good catch - thanks.